src/routes/dbinitRoute.js

```diff
@@ -0,0 +1,162 @@
+const { Router } = require('express');
+const City = require('../models/City');
+const Country = require('../models/Country');
+const Actor = require('../models/Actor');
+const Genre = require('../models/Genre');
+
+const router = Router();
+
+router.get('/dbinit', async (req, res) => {
+  const modelsArray = [City, Country, Actor, Genre];
+
+  // db emptiness check
+  async function emptyCheck(models) {
+    let isEmpty = true;
+    for (let i = 0; i < models.length; i++) {
+      const result = await models[i].findOne({});
+      if (result) {
+        isEmpty = false;
+        return isEmpty;
+      }
+    }
+    return isEmpty;
+  }
+
+  if (await emptyCheck(modelsArray)) {
+    // Countries
+    const belarus = new Country({
+      name: 'Belarus',
+    });
+    const ukraine = new Country({
```

Comment:
it's better to get **countries, cities, actors, genres** from a different .js file and loop over them filling db. 
this solution will remove 100 lines of code and make it more readable.